CONTRIBUTING.md

```diff
@@ -141,7 +141,7 @@ To debug e2e tests we can use following run configuration
 
 In following snippet:
 1. Adjust program to the test you want to run
-2. Review MCLI_E2E_BINARY to point to mongodbcli or atlas depending on what CLI you want to test.
```

Comment:
https://github.com/mongodb/mongodb-atlas-cli/blob/35ee197eda86bb657cd0dc9c7fac4c14aec42088/test/e2e/cli.go#L25-L49

Also pointing MCLI_E2E_BINARY to the atlas bin may be dangerous for iam commands as they will assume an `iam` prefix alas `mongocli iam projects` where for atlas it'll expect `atlas projects`

Reply:
Clarified that. We have learned that the hard way yesterday :)

Reply:
it shouldn't matter at come point I considered making `cli.Bin()` env aware but then I hit the iam wall making it not a straight forward change

At the same time in the current state of the codebase it doesn't matter which bin you are using because the node (the cobra command) will be the same for both binaries and we get away with it

If using the `mongocli` bin for the SDK migration is a problem then for the project the comamnds will have to be duplicated, one set for atlas bin and one set for mongocli

Reply:
> If using the mongocli bin for the SDK migration is a problem then for the project the commands will have to be duplicated, one set for atlas bin and one set for mongocli

CC @swattyT @matt-condon - very important comment.

Reply:
and to be clear from where I see it using the `mongocli` bin should be a non-issue, it's just an entry point and it makes no difference if it's called `mongocli`, `atlas` or `cly-mccliface`, at the end it calls the same logiic